### PR TITLE
FUSETOOLS2-872 - fix unstable UI tests

### PR DESCRIPTION
**Pull request by mlorinc**
Change log:

- native dialog is no longer used
- folder is opened for each generated project (it fixes problems with Windows file lock)
- added input checks to increase stability
- replaced ExtensionViewSection with Marketplace class from vscode-uitests-tooling
- changed test timers
- changed test case titles to be more file name friendly

Signed-off-by: Marian Lorinc <mlorinc@redhat.com>

### CODE REVIEW

File: .mocharc.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "bail": true
```
Comment (djelinek) [inline .mocharc.json:2]: is this change really needed? it looks like `true` is default value for bail (see https://mochajs.org/api/mocha#bail)

File: .vscode/settings.json [hunk 2]
```diff
@@ -6,4 +6,4 @@
     "search.exclude": {
         "out": true // set this to false to include "out" folder in search results
```
Comment (djelinek) [inline .vscode/settings.json:7]: why is this change needed?

**Review by djelinek — CHANGES_REQUESTED** [review]

File: .mocharc.json (hunk 1 above)
Comment (mlorinc) [reply .mocharc.json]: No, it is not.

File: .vscode/settings.json (hunk 2 above)
Comment (mlorinc) [reply .vscode/settings.json]: It is not needed. My editor inserts final EOL when saving files. I have removed it.

**Review by djelinek — APPROVED** [review]

**Review by lhein — APPROVED** [review]

**Comment by lhein** [comment]
I merged some other PRs meanwhile, Marian. Please rebase this one.

**Review by bfitzpat — APPROVED** [review]
